node/src/chain_spec/mod.rs

```diff
@@ -103,3 +103,61 @@ pub fn validate_allocation(
 	);
 	assert_eq!(total_allocated, total_tokens, "total allocated does not equal the desired amount");
 }
+
+/// Validate that the allocated fits the following criteria:
+/// - no duplicate timestamps
+/// - no duplicate accountIds per timestamp
+/// - total amount allocated is correct
+/// - times are within a given range
+pub fn validate_vesting(
+	allocated_accounts: Vec<(AccountId, Balance)>,
```

Comment:
why is this in here?

Reply:
cause i want to test the total amount of allocated + vesting together. or should i just test vesting by itself?

Reply:
chatted on slack - should be separate, since separate functionality. fixed